color-eyre/src/section/help.rs

```diff
@@ -289,6 +327,20 @@ impl Display for HelpInfo {
                     write!(indented(f).ind(n), "{}", error.style(theme.help_info_error))?;
                 }
 
+                Ok(())
+            }
+            HelpInfo::Report(report, theme) => {
+                let chain_errors = report.chain();
+                write!(f, "Report:")?;
+                for (n, error) in chain_errors.enumerate() {
+                    writeln!(f)?;
+                    write!(
+                        indented(f).ind(n),
+                        "{}",
+                        error.style(theme.help_info_report)
+                    )?;
+                }
```

Comment:
If you have any backtrace or spantrace information in this report, it won't be printed. That feels like a motivating use case for this feature

Reply:
Hello @vultix , thank you very much for the feedback!

If it's not too much trouble, could you explain in more detail what seems to be wrong?

When running the examples I added, the backtrace information appears to be displayed correctly, very similar to how it is shown when running the error examples:
![image](https://github.com/user-attachments/assets/0c05fe72-f731-4028-a65b-a8e275ed9171)

I would really appreciate any clarification you can provide.

Reply:
The motivation in #198 (by me) was "render [...] including helpful sections like from `wrap_err` or `with_suggestion`".

Your screenshot does show one backtrace. It's for the parent error, which is regular behavior, unaffected by this PR.
But there should be three backtraces shown. The two "Report" sections at the bottom, the actual new feature, only show an error message, when they should show all their `HelpInfo` sections. That's what would set this apart from the existing `with_error`.